cypress/integration/collections/view_page_spec.js

```diff
@@ -101,6 +101,23 @@ describe("Collections View page", () => {
       });
     });
 
+    it("should display URL params pagination", () => {
+      cy.get(".rs-pagination").within($pagination => {
+        cy.get("a")
+          .eq(5)
+          .click();
+        // Open page 5 of results
+        cy.location("search").should("include", "?collection-items-results=5");
+        // Visit an item detail page
+        cy.visit("/items/acec9f18-a2aa-424c-b0b8-fcaaaf579ba0", {
+          timeout: 3000
+        });
+        // Go back through browsers history and expect to get to page 5 of results.
+        cy.go("back");
+        cy.location("search").should("include", "?collection-items-results=5");
```

Comment:
Can we also add a visual check here.  Like 5 is highlighted, or active, or whatever that state is, in the pagination links?